tests/mshevchuk/homework_7_retry.py

```diff
@@ -0,0 +1,31 @@
+"""
+Create decorator, which will retry inner function multiple times untill it passes
+"""
+
+import random
+
+
+def retry_unstable_function(func):
+    def retry():
```

Comment:
let function receive any amount of position and named parametes, so that you could reuse it